### PR TITLE
Added mixin to generate panel indicator

### PR DESCRIPTION
**Pull request by MrDinsdale**
## Description
Extracts panel indicator from tiles so it can be used on other elements such as buttons to signify a relationship with the associated panel.

## Related Issue
https://github.com/sky-uk/toolkit-ui/issues/114

## Motivation and Context
Why is this change required? What problem does it solve?

## How Has This Been Tested?
Tested in local dev environment

## Screenshots (if appropriate)
screen shot 2016-11-10 at 09 43 27

## Types of Changes
What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Browser Support
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE9
- [ ] IE10
- [ ] IE11
- [ ] Opera
- [x] Safari
- [ ] Mobile Devices

## Checklist
- [x] New functions and mixins have appropriate tests.
- [x] All new and existing tests passed.
- [ ] I have added instructions on how to test my changes.
- [x] Package version updated.
- [x] CHANGELOG.md updated.

### CODE REVIEW

**Comment by MrDinsdale** [comment]
Decided this would sit better in toolkit-ui within the panel component.

**Comment by MrDinsdale** [comment]
Currently `sass-true` doesn't full return the styling set by complex mixins featuring pseudo elements.

The tests I've added check that the correct theme is applied however not the full mixin output.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Toolkit Core v1.1.0
+
+## 1. Features
+- [mixins-ui] `panel-indicator` extracted from tile to a mixin so can now be used on any elements.
```
Comment (joebell93) [inline CHANGELOG.md]: Feel like this sentence is a bit jumbled:

Could we change to:
```
- [mixins-ui] `panel-indicator` extracted from tile into a mixin for use on any element.
```

Maybe?

File: tools/_mixins-ui.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
+// =============================================================================
+// TOOLS / MIXINS UI
+// =============================================================================
+
+// Panel Indicator
+// ===========================================
+
+// Adds panel indicator to show relationship with element that opened it.
```
Comment (joebell93) [inline tools/_mixins-ui.scss]: Could we include an example of it's usage here?
Something I'm keen to improve in our CSS docs going forwards.

e.g.
https://github.com/sky-uk/toolkit-ui/blob/master/components/_select.scss#L31

File: tools/_mixins-ui.scss [hunk 3]
```diff
@@ -0,0 +1,86 @@
+// =============================================================================
+// TOOLS / MIXINS UI
```
Comment (joebell93) [inline tools/_mixins-ui.scss]: I'm wondering if maybe we should either:

a) Move UI mixins into here (e.g. divider)
b) Rename this to be it's own file instead? (panel)

Thoughts?

Comment (MrDinsdale) [inline tools/_mixins-ui.scss]: I was thinking the same however not sure it should be with in this commit perhaps?

We could do another PR for that once this is in and include in the same release. Just so we have things nicely isolated.
